doc/devel/testing.rst

```diff
@@ -151,26 +140,6 @@ See the documentation of `~matplotlib.testing.decorators.image_comparison` and
 `~matplotlib.testing.decorators.check_figures_equal` for additional information
 about their use.
 
-Known failing tests
--------------------
-
-If you're writing a test, you may mark it as a known failing test with the
-:func:`pytest.mark.xfail` decorator. This allows the test to be added to the
-test suite and run on the buildbots without causing undue alarm. For example,
-although the following test will fail, it is an expected failure::
-
-  import pytest
-
-  @pytest.mark.xfail
-  def test_simple_fail():
-      '''very simple example test that should fail'''
-      assert 1 + 1 == 3
-
-Note that the first argument to the :func:`~pytest.mark.xfail` decorator is a
-fail condition, which can be a value such as True, False, or may be a
-dynamically evaluated expression. If a condition is supplied, then a reason
-must also be supplied with the ``reason='message'`` keyword argument.
```

Comment:
This is a somewhat lengthy description of a standard pytest feature. Additionally, you will in general not write tests that are expected to fail. So this section is more distracting than helpful.